salesforce-marketing-cloud/3.2/modules/ROOT/pages/salesforce-mktg-connector-reference.adoc

```diff
@@ -568,282 +652,279 @@ Upserts one or more API objects. IMPORTANT: When you map your objects to the inp
+
+Configures how to identify a redelivered message and how to find out when the message was redelivered.
 
 [%header,cols="20s,25a,30a,15a,10a"]
 |===
 | Field | Type | Description | Default Value | Required
-| Path a| String | The path to the CRL file. |  |
+| Use Secure Hash a| Boolean | If `true`, Mule uses a secure hash algorithm to identify a redelivered message. |  |
+| Id Expression a| String | One or more expressions that determine when a message was redelivered. You can set this property only if the *Use Secure Hash* field is `false`. |  |
 |===
 
-[[Reconnection]]
-=== Reconnection
+[[RepeatableFileStoreIterable]]
+=== Repeatable File Store Iterable
+
+Configures the repeatable file store iterable type.
 
 [%header,cols="20s,25a,30a,15a,10a"]
 |===
 | Field | Type | Description | Default Value | Required
-| Fails Deployment a| Boolean | When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. |  |
-| Reconnection Strategy a| * <<reconnect>>
-* <<reconnect-forever>> | The reconnection strategy to use. |  |
-|===
-
-[[reconnect]]
-=== Reconnect
+| In Memory Objects a| Number | Maximum amount of instances to keep in memory. If more than that is required, content on the disk is buffered. |  |
+| Buffer Unit a| Enumeration, one of:
 
-[%header%autowidth.spread]
-|===
-| Field | Type | Description | Default Value | Required
-| Frequency a| Number | How often in milliseconds to reconnect. | |
-| Count a| Number | How many reconnection attempts to make. | |
-| blocking |Boolean |If false, the reconnection strategy runs in a separate, non-blocking thread. |true |
+** BYTE
+** KB
+** MB
+** GB | Unit for the *In Memory Objects* field. |  |
 |===
 
-[[reconnect-forever]]
-=== Reconnect Forever
+[[RepeatableInMemoryIterable]]
+=== Repeatable In Memory Iterable
 
-[%header%autowidth.spread]
+Configures the repeatable in memory iterable type.
+
+[%header,cols="20s,25a,30a,15a,10a"]
 |===
 | Field | Type | Description | Default Value | Required
-| Frequency a| Number | How often in milliseconds to reconnect. | |
-| blocking |Boolean |If false, the reconnection strategy runs in a separate, non-blocking thread. |true |
+| Initial Buffer Size a| Number | Amount of instances to initially keep in memory to consume the stream and provide random access to it. If the stream contains more data than can fit into this buffer, then the buffer expands according to the *Buffer Size Increment* attribute, with an upper limit of *Max In Memory Size*. Default value is 100 instances. |  |
+| Buffer Size Increment a| Number | This is by how much the buffer size expands if it exceeds its initial size. Setting a value of `0` or lower means that the buffer must not expand, meaning that a `STREAM_MAXIMUM_SIZE_EXCEEDED` error is raised when the buffer gets full. Default value is 100 instances. |  |
```

Comment:
In the spreadsheet:

Amount by which the buffer size expands if it exceeds its initial size. Setting a value of `0` or lower specifies that the buffer can't expand. 

You can add the next sentence. In terms of the default, I would add it in the last column and remove the text from here.